ui/app/AppLayouts/CommunitiesPortal/CommunitiesPortalLayout.qml

```diff
@@ -34,164 +35,241 @@ StatusSectionLayout {
         id: d
 
         property string searchText: ""
-        property int layoutVMargin: 70
-        property int layoutHMargin: 64
-        property int titlePixelSize: 28
-        property int subtitlePixelSize: 17
+
+        // values from the design
+        readonly property int layoutTopMargin: 10
+        readonly property int layoutBottomMargin: 249
+        readonly property int layoutHMargin: 64
+        readonly property int layoutWidth: 1037
+        readonly property int scrollViewTopMargin: 20
+
+        readonly property int titlePixelSize: 28
+        readonly property int subtitlePixelSize: 17
+
+        readonly property int preventShadowClipMargin: 40
 
         function navigateToCommunity(communityId) {
             root.communitiesStore.setActiveCommunity(communityId)
         }
+
+        readonly property bool searchMode: searcher.text.length > 0
+    }
+
+    SortFilterProxyModel {
+        id: featuredModel
+
+        sourceModel: root.communitiesStore.curatedCommunitiesModel
+
+        filters: ValueFilter {
+            roleName: "featured"
+            value: true
+        }
+    }
+
+    SortFilterProxyModel {
+        id: popularModel
+
+        sourceModel: root.communitiesStore.curatedCommunitiesModel
+
+        filters: ValueFilter {
+            roleName: "featured"
+            value: false
+        }
+    }
+
+    SortFilterProxyModel {
+        id: searchModel
+
+        sourceModel: root.communitiesStore.curatedCommunitiesModel
+
+        filters: ExpressionFilter {
+            enabled: searcher.text.length > 0
+            expression: {
+                searcher.text
+                return name.toLowerCase().includes(searcher.text.toLowerCase())
+            }
+        }
     }
 
     centerPanel: Item {
-        implicitWidth: parent.width
-        implicitHeight: parent.height
+        anchors.top: parent.top
+        anchors.bottom: parent.bottom
         anchors.left: parent.left
+        width: d.layoutWidth
+
+        anchors.topMargin: d.layoutTopMargin
```

Comment:
could you please check that all content is aligned with section header before adding top margins? this job was done [here](https://github.com/status-im/status-desktop/pull/7342/files#diff-4d4f432ed9442e3f1baa07349d2da11d741b4aeea46523f4ff28a06b307ef190) so please make sure no regression is introduced

Reply:
I'm matching margins from this design: https://www.figma.com/file/17fc13UBFvInrLgNUKJJg5/Kuba%E2%8E%9CDesktop?node-id=8159%3A415655. So I presume that it's ok.